### DISCUSSION TITLE
Labeling Key Points on Mice

### DISCUSSION THREAD

**Discussion by jmdelahanty** [talmolab/sleap#805] [General]
Hey everyone!

I was wondering what people using top-down camera views (1 camera) should do for labeling body parts that aren't visible much or most of the time.

For example, if you have a camera view where you can't see the forelimbs most of the time, should you prioritize/label only structures that you can see pretty much constantly? Sometimes you don't care what their paws are doing, so it's unnecessary to label them I'd imagine, but I guess I was wondering what you would recommend.

**Comment by talmo** [comment]
Hi @jmdelahanty,

In general you'll want to pick the nodes that are most visible since labeling ones that are often occluded will both be laborious and unreliable.

Check these out also:
- [Guide](https://sleap.ai/guides/skeletons.html): Tips on skeleton design.
- [Datasets](https://sleap.ai/datasets.html): See the mouse datasets we provide for reference on skeletons in different conditions.

Cheers,

Talmo

**Reply by jmdelahanty** [reply]
Thanks for the advice Talmo!
